### DISCUSSION TITLE
لینک های اتصال بعد از چند ساعت خود به خود از کار میافته و با اضافه کردن مجدد همون لینک مشکل حل میشه بعد چند ساعت دوباره.... لطفا راهنمایی کنید

### DISCUSSION THREAD

**Discussion by hdnteam** [hiddify/Hiddify-Manager#900] [Q&A سوالات رایج]
سلام لطفا راهنمایی کنید 
 لینک های اتصال بعد از چند ساعت خود به خود تغییر می کنه و همه قطع میشن لطفا راهنمایی کنید
سرور مجدد نصب کردم ریبیلد زدم بک آپ انداختم نشد
همه کلافه شدیم هی لینک سرور ها از کار میافته
ضمنا تونل استفاده میکنم
سه تا دامین relay اضافه کردم تونل از دیتا سنتر های مختلف

**Comment by drsat2006** [comment]
سلام .در صورتی که مشکلتون رو حل نکردید و تمام موارد لازم برای ایجاد سرور را دارید .با بنده در تماس باشید .به من پیام بدید مشکلتون رو حل میکنم.
در صوتی که با اهنمایی نتونستید درست کنین کار رو .دسترسی به سرور و دامنه ها بدید براتون همه چی رو نصب شده تویل میدم.فقط چون سرم بسیار شلوغه یکم طول میکشه تا پیام رو ببینم و مجدد جواب بدم. مقداری صبر باید برای کارتون داشته باشید .
فقط در صورتی با بنده تماس بگیرید که همه راه ها را رفته باشید و توانایی صبر را داشته باشید . 
بنده برنامه نویس بورس هستم . ولی برای کمک به شما اگه خواستید کارتون رو میتونم انجام بدم .دامنه معتبر و سرور با ایپی تمیز اگر دارید . و همه کارها رو انجام دادید و نتیجه نگرفتید با بنده تماس بگیرید .
